src/APIFunctions/Mailer.js

```diff
@@ -22,3 +22,46 @@ export async function sendVerificationEmail(email, firstName) {
     });
   return status;
 }
+
+/**
+ * Verify a user's email after the visiting the link sent to their email.
+ * @param {string} email - The user's email
+ * @param {string} hashedId - A hashed value of the user's mongoDB _id field
+ * @returns {ApiResponse} Containing any error information related to the
+ * request
+ */
+export async function validateVerificationEmail(email, hashedId) {
+  let status = new ApiResponse();
+  await axios
+    .post(MAILER_API_URL + '/api/mailer/validateVerificationEmail', {
+      email,
+      hashedId
+    })
+    .catch(err => {
+      status.responseData = err;
+      status.error = true;
+    });
+  return status;
+}
+
+/**
+ * Invokes the google spreadsheet API to add officer application form data to it
+ * @param {object} row - the data from the officer app form
+ * @param {String} sheetsID - ID of the google sheet getting updated
+ * @returns {ApiResponse} Containing any error information related to the
+ * request
+ */
+export async function addToSpreadsheet(row, sheetsID){
```

Comment:
anything google sheets related should go into a new file in `APIFunctions` like `APIFunctions/GoogleSheets.js`